datarepo-clienttests/src/main/java/utils/KubernetesClientUtils.java

```diff
@@ -352,19 +350,40 @@ public static void changeReplicaSetSizeAndWait(int podCount) throws Exception {
 
   private static long printApiPodCount(V1Deployment deployment, String message)
```

Comment:
Nit: I think it's slightly cleaner to replace calls to printApiPodCount with logger.debug("Pod Count: {}; Message: {}", getApiPodCount, message). Just because this print method isn't adding much value besides log formatting, now that the work has been moved to getApiPodCount, which I think is great!

Reply:
Sounds good!